test/system/src/test/java/io/pravega/test/system/MultiReaderWriterWithFailOverTest.java

```diff
@@ -338,7 +338,7 @@ private void performFailoverTest() throws InterruptedException {
         //Scale down SSS instances to 2
         segmentStoreInstance.scaleService(2, true);
         //zookeeper will take about 60 seconds to detect that the node has gone down
```

Comment:
We could do something like this `EventWriterConfig.builder().retryAttempts(10).maxBackoffMillis(10000).build()`
Max Backoff delay is 10 seconds